auth/authenticate.js

```diff
@@ -0,0 +1,65 @@
+const jwt = require("jsonwebtoken");
+
+const jwtKey = process.env.JWT_SECRET;
+
+module.exports = {
+  authenticate,
+  generateToken,
+  requireAdmin
+};
+function authenticate(req, res, next) {
```

Comment:
great job using JWT's for authentication, the requireAdmin aspect satisfies rubric requirements for 3 stars on the authentication objective.  Well done